### PR TITLE
#237 Reduce docker reliance in PIXL-EHR

### PR DESCRIPTION
**Pull request by peshence**
- remove unused rabbitmq instance
- add docker compose to testconf
- remove run_tests scripts

Still needs docker because it tests whether something has happened in the DB. Not familiar enough with things to tell if that's necessary. 

But if it is, perhaps we could make a dev container with all the test infrastructure running locally? That way we don't have to keep turning docker on and off for each test run

### CODE REVIEW

File: pixl_ehr/tests/docker-compose.yml [hunk 1]
```diff
@@ -58,10 +46,3 @@ services:
         N_TABLE_ROWS: 2
     ports:
       - "35433:5432"
-
-    healthcheck:
-      test: ["CMD", "pg_isready", "-U", "postgres"]
-      start_period: 30s
-      interval: 10s
-      timeout: 30s
-      retries: 5
```
Comment (milanmlft) [inline pixl_ehr/tests/docker-compose.yml:67]: Why remove only the healthcheck? If we keep the postgres database, would probably still need this no?

**Comment by milanmlft** [comment]
Nice! I think regarding the PIXL postgres database, we might also be able to mock it instead of having to spin up the Docker service? But not sure how feasible that is.
I definitely like the idea of a devcontainer.

**Comment by peshence** [comment]
> Nice! I think regarding the PIXL postgres database, we might also be able to mock it instead of having to spin up the Docker service? But not sure how feasible that is. I definitely like the idea of a devcontainer.

I would generally agree about mocking when it comes to unit tests. However, these are not unit tests - they are functional black box tests, in the sense that they don't test specific pieces of logic but rather the function of the module (and really its communication with other modules, dbs, etc).

**Review by stefpiatek — APPROVED** [review]
Nice, for now agree keeping the postgres in is fine. Ultimately we'll aim to not interact with postgres at all and this will be an exporting api

**Pull request merged**